codegen/templates/field.gotpl

```diff
@@ -38,7 +38,7 @@
 			Field: field,
 		}
 		ctx = graphql.WithResolverContext(ctx, rctx)
-		resTmp := ec.FieldMiddleware(ctx, {{if $object.Root}}nil{{else}}obj{{end}}, func(ctx context.Context) (interface{}, error) {
+		ctx, resTmp := ec.FieldMiddleware(ctx, {{if $object.Root}}nil{{else}}obj{{end}}, func(ctx context.Context) (interface{}, error) {
```

Comment:
Its probably preferable to update ctx rather than shadowing it and disabling a bunch of linter checks?

Reply:
It sees that there is not much value for the this hard work... 🤔